README.md

```diff
@@ -95,3 +95,5 @@ g
 **213.** If the rules are changed so that further play is impossible, or if the legality of a move cannot be determined with finality, or if by the Judge's best reasoning, not overruled, a move appears equally legal and illegal, then the first player unable to complete a turn is the winner.
 
 This rule takes precedence over every other rule determining the winner. 
+
+**302.** Repeal of rule 204. Voting AGAINST a rule does not incur any penalties or reward any benefits. To promote additional rules (and thus playing the game), voting against a rule should not be rewarded. Gaining points by voting AGAINST is unconducive to the game and should be discouraged.
```

Comment:
The first sentence is sufficient for defining the rule, wile the next two sentences explain the spirit of the rule. 

**Question to everyone:** do we find such explanations useful to have in a rule or should we keep rules as brief and concise as possible?

Reply:
As you said, it is to explain the spirit of the rule, so that when in the future there is discussion on whether it should repealed we know what the intent of it was so to better determine its value.

As to why repealing 204 rather than 206? Frankly 206 did not occur to me, but that is a good one for another round.